pom.xml

```diff
@@ -301,47 +313,29 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <source>8</source>
+                    <source>11</source>
                 </configuration>
             </plugin>
 
+            <!-- https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>${maven-checkstyle-plugin.version}</version>
-                <dependencies>
-                    <dependency>
-                        <groupId>com.labs64.commons</groupId>
-                        <artifactId>code-analysis</artifactId>
-                        <version>${code-analysis.version}</version>
-                    </dependency>
-                </dependencies>
```

Comment:
Why is this removed? Shouldn't it deliver config files for the respective plugins?

Reply:
`code-analysis` is an internal Labs64 artefact and isn't available in public.
As the Gateway module can be maintained by the external contributors, this will most likely cause a build error.

Also, delivering `SNAPSHOT` isn't the best practice.

Reply:
Probably, we need to include quality rules/metrics for the plugins in the Gateway project itself.

Reply:
Labs64 dependency replaced by code style definitions